src/apps/bridge/bridgePowerController.cpp

```diff
@@ -123,6 +123,9 @@ void BridgePowerController::powerBusAndSetSignal(bool on, bool notifyL2) {
     xEventGroupSetBits(_busPowerEventGroup, signal_to_set);
     if (notifyL2) {
       bm_l2_netif_set_power(on);
+      if (on) {
+        bm_l2_netif_enable_disable_port(2, !on);
```

Comment:
I think I'd prefer an explicit `false` instead of `!on`.

I think it would also be helpful to document some implicit tribal knowledge with a comment on the preceding line(s) explaining to our future colleagues and to the open source community that bridge port 2 is not connected to anything, so we should always save the power by disabling it.